ocaml/xapi/records.ml

```diff
@@ -229,8 +229,8 @@ let network_sriov_record rpc session_id network_sriov =
     fields =
       [
         make_field ~name:"uuid"                     ~get:(fun () -> (x ()).API.network_sriov_uuid) ();
-        make_field ~name:"physical-pif-uuid"        ~get:(fun () -> get_uuid_from_ref (x ()).API.network_sriov_physical_PIF) ();
-        make_field ~name:"logical-pif-uuid"         ~get:(fun () -> get_uuid_from_ref (x ()).API.network_sriov_logical_PIF) ();
+        make_field ~name:"physical-PIF"        ~get:(fun () -> get_uuid_from_ref (x ()).API.network_sriov_physical_PIF) ();
```

Comment:
The old one is better: in the CLI, we always have `*-uuid`.

Reply:
But the corresponding fields in `vlan_record` and `tunnel_record` and `bond_record` don't have the `-uuid` suffix.

Reply:
Hmm, I see... so it is inconsistent already :(